web_sdk/BUILD.gn

```diff
@@ -286,6 +289,7 @@ prebuilt_dart_action("flutter_dartdevc_canvaskit_html_kernel_sdk") {
     "--libraries-file",
     "org-dartlang-sdk:///flutter_web_sdk/libraries.json",
     "--inline-source-map",
+    "-DFLUTTER_WEB_USE_SKIA=true",
```

Comment:
Does autodetect use skia=true?

Reply:
You can remove this. It doesn't matter once FLUTTER_WEB_AUTO_DETECT is specified.

Reply:
autodetect has higher precedence, it doesn't check skia at that point.

Reply:
OK, I'll leave it as set just so it is easier to change in the future.